backend/src/models/entry/entryAnalysis.ts

```diff
@@ -74,32 +77,49 @@ entryAnalysisSchema.methods.getAnalysisContent = async function (configId: strin
       }
     }
   }
+  return config.model.analysis;
+}
 
-  const cdGpt = new CdGpt(process.env.OPENAI_API_KEY, config.model.analysis);
+async function getAnalysisCompletion(configModelAnalysis: string, content: string): Promise<any> {
+  if (process.env.OPENAI_API_KEY === undefined) {
+    throw new Error('OPENAI_API_KEY undefined. Must be set to get analysis')
```

Comment:
Lint error: Missing semicolon.

```suggestion
    throw new Error('OPENAI_API_KEY undefined. Must be set to get analysis');
```